scavenger-collector/src/main/kotlin/com/navercorp/scavenger/repository/AgentStateDao.kt

```diff
@@ -68,4 +68,9 @@ class AgentStateDao(
                 .addValue("ids", ids)
         )
     }
+
+    fun toInstantMicro(instant: Instant): Instant {
+        val epochMicro = (instant.epochSecond * 1_000_000) + (instant.nano / 1000)
+        return Instant.ofEpochSecond(epochMicro / 1_000_000, (epochMicro % 1_000_000) * 1000)
+    }
```

Comment:
After switching to jdk17, when using Instant.now(), I modified it to use micro because it tends to use different time precision depending on the OS.
That's why the test was previously crashed.

https://stackoverflow.com/questions/74781495/changes-in-instant-now-between-java-11-and-java-17-in-aws-ubuntu-standard-6-0

Reply:
that's new to me, thanks 😄